mail_restrict_follower_selection/models/mail_followers.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2018 Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class MailFollowers(models.Model):
+    _inherit = 'mail.followers'
+
+    @api.model
+    def _mail_restrict_follower_selection_get_domain(self):
```

Comment:
any subtle difference I'm missing, or did you just copy the function from https://github.com/etobella/social/blob/0b4d43ae4de03c2003aad1ceac107858c8ca095c/mail_restrict_follower_selection/models/mail_wizard_invite.py#L13 - if so, just call it instead

Reply:
@hbrunn all changes have been applied. Also, I detected an error on https://github.com/etobella/social/blob/0b4d43ae4de03c2003aad1ceac107858c8ca095c/mail_restrict_follower_selection/models/mail_wizard_invite.py#L13 , as it was missing sudo when reading ir.config_parameter. It is fixed now.